src/transcoder/transcoderoptionsdialog.cpp

```diff
@@ -55,11 +56,14 @@ TranscoderOptionsDialog::TranscoderOptionsDialog(Song::FileType type,
       options_ = new TranscoderOptionsWma(this);
       break;
     default:
+      ui_->errorMessage->setText(
+          tr("Unknown encoder type: %1").arg(preset.name_));
       break;
   }
 
   if (options_) {
-    setWindowTitle(windowTitle() + " - " + Song::TextForFiletype(type));
+    ui_->errorMessage->setVisible(false);
+    setWindowTitle(windowTitle() + " - " + Song::TextForFiletype(preset.type_));
```

Comment:
We should fix the translation of this, ideally.

Reply:
Fixed. I think this is what you had in mind?